options.html

```diff
@@ -30,11 +30,25 @@
 							<br />
 							<input type="checkbox" id="minihelp" name="minihelp">
 							Show mini help
+							<br />
+							<br />
+							Show popup with delay: 
+							<input type="number" id="popupDelay" name="popupDelay" min="1"> milliseconds
 							</p>
 					</div>
 					<!-- <div id="keycon" class="tabcontent hide"> -->
 					<div id="keycon" class="tabContent">
 					<h1>Keyboard</h1>
+						<input type="checkbox" id="showOnKeyFlag" name="showOnKeyFlag" disabled>
+						Immediately show only when key <select id="showOnKey" name="showOnKey"> 

```

Comment:
My reasoning was: user has to make _some minimal action_ to show popup. For usual mode I hover mouse and hold it over a word for at least 150ms. But in other mode I'm already doing _some action_ (keeping button pressed), so I expect app response immediately.

But it doesn't matter anymore, I'll make options separate.